rfc6962-bis.xml

```diff
@@ -588,11 +587,31 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
       <section title="Merkle Tree Head" anchor="tree_head">
         <figure>
           <preamble>
-            The log stores information about its Merkle Tree in a <spanx style="verb">TransItem</spanx> structure of type <spanx style="verb">tree_head_v2</spanx>, which encapsulates a <spanx style="verb">TreeHeadDataV2</spanx> structure:
+            The log may store information about its Merkle Tree in a <spanx style="verb">TreeHeadDataV2</spanx>:
           </preamble>
           <artwork>
     opaque NodeHash&lt;32..2^8-1&gt;;
 
+    enum {
+        reserved(65535)
+    } SthExtensionType;
+
+    struct {
+        SthExtensionType sth_extension_type;
+        opaque sth_extension_data&lt;0..2^16-1&gt;;
+    } SthExtension;</artwork>
+        </figure>
+        <t>
+          The length of NodeHash MUST match HASH_SIZE of the log.
+        </t>
+        <t>
+          <spanx style="verb">sth_extension_type</spanx> identifies a single extension from the IANA registry in <xref target="sth_extension_types"/>. At the time of writing, no extensions are specified.
+        </t>
+        <t>
+          The interpretation of the <spanx style="verb">sth_extension_data</spanx> field is determined solely by the value of the <spanx style="verb">sth_extension_type</spanx> field. Each document that registers a new <spanx style="verb">sth_extension_type</spanx> must describe how to interpret the corresponding <spanx style="verb">sth_extension_data</spanx>.
+        </t>
+        <figure>
```

Comment:
I meant the order of the explanations of these fields, not the fields themselves.  Just for consistency.  Only a nit though.